packages/admin/admin/src/common/readOnlyField/ReadOnlyDateField.tsx

```diff
@@ -0,0 +1,66 @@
+import { Calendar, Lock } from "@comet/admin-icons";
+import { css } from "@mui/material/styles";
+import { FunctionComponent, ReactNode } from "react";
+
+import { createComponentSlot } from "../../helpers/createComponentSlot";
+import { ReadOnlyFieldLabel } from "./ReadOnlyFieldLabel";
+
+export const ReadOnlyDateField: FunctionComponent<{
+    label?: ReactNode;
+    value?: Date | string | undefined;
+    className?: string;
+}> = ({ label, value, className }) => {
+    const date = value instanceof Date ? value : value ? new Date(value) : undefined;
```

Comment:
For dates without time we use string (e.g., `2025-03-06`). This field should work with strings as well.